### DISCUSSION TITLE
Docs code block standards

### DISCUSSION THREAD

**Discussion by shaedrich** [laravel/framework#53244] [Q&A]
When checking code block contents in the docs, I saw a lot of inconsistencies:
* Should the `<?php` be included or omitted? (https://github.com/laravel/framework/discussions/54661)
* Should the `class ClassName { … }` be included or omitted when describing a method?
* Should the `namespace` be included or omitted? (https://github.com/laravel/framework/discussions/54662)
* Should the `use` statement for used classes, traits, and interfaces be included or omitted? (https://github.com/laravel/framework/discussions/54663)
* Is [highlighting](https://torchlight.dev/docs/annotations/highlighting), [focusing](https://torchlight.dev/docs/annotations/focusing), or [collapsing](https://torchlight.dev/docs/annotations/collapsing) preferred? (https://github.com/laravel/framework/discussions/54664)
* Semicolon after last statement?
* \`\`\` or indent? (https://github.com/laravel/docs/pull/10172)

**Comment by ghost** [comment]
@shaedrich I see you want to clean up and put stuff in order:)

**Reply by shaedrich** [reply]
If that's not vetoed 😅

Especially, Torchlight brings some features that might not have been available when some code samples were originally written.

**Reply by shaedrich** [reply]
Apparently, this won't be necessary anymore since [Taylor decided to rework the docs](https://github.com/laravel/laravel.com/pull/366#issuecomment-2450483815) and reject changes [he himself requested](https://github.com/laravel/docs/pull/9972#issuecomment-2427577765). How the rework will look, probably only he knows, so we'll see.

**Comment by Sophist-UK** [comment]
As a coder I prefer to have things consistent in my editor - but the choice of which of the current multiple ways is also important, and personally I would prefer to have things inconsistent if the choice of way is one I absolutely dislike (better to have some that are OK rather than none that are OK).

It seems to me that @shaedrich Sebastian is looking for input from the community here on what choices to narrow down to. And personally, for the sake of absolute clarity, I would like to see a detailed comment here giving examples of the choice to narrow to for each of the inconsistencies. Sebastian, is this something that you can provide?

**Reply by shaedrich** [reply]
Apparently, this won't be necessary anymore since [Taylor decided to rework the docs](https://github.com/laravel/laravel.com/pull/366#issuecomment-2450483815) and reject changes [he himself requested](https://github.com/laravel/docs/pull/9972#issuecomment-2427577765). How the rework will look, probably only he know, so we'll see.

**Reply by shaedrich** [reply]
@Sophist-UK I added polls with examples to the post above

**Reply by shaedrich** [reply]
fyi: The rework is done (laravel/docs#10188), so let's see where this goes
